dojo/fixtures/test_type.json

```diff
@@ -606,18 +606,25 @@
 		"model": "dojo.test_type",
 		"pk": 178
 	},
-  {
+	{
 		"fields": {
 		  "name": "Yarn Audit Scan"
     },
 		"model": "dojo.test_type",
 		"pk": 179
 	},
 	{
-    "fields": {
-      "name": "ESLint Scan"
-    },
-    "model": "dojo.test_type",
-      "pk": 180
+		"fields": {
+			"name": "ESLint Scan"
+		},
+		"model": "dojo.test_type",
+		"pk": 180
+	},
+	{
+		"fields": {
+		  "name": "AWS Security Hub Scan"
+		},
+		"model": "dojo.test_type",
+		"pk": 181
```

Comment:
Will require update, as other importers got merged in between.

Reply:
Updated.